servers/database/src/main/resources/db/migration/maps_db/V1.00.08__maps_repo_store_last_comit_date.sql

```diff
@@ -0,0 +1,12 @@
+/* recreate map_index table with 'last_commit_date' column rather than a 'version' column. */
+drop table map_index;
+create table map_index
+(
+    id           serial primary key,
+    map_name     varchar(256) not null,
+    last_commit_date      timestamp not null check(last_commit_date < now()),
```

Comment:
I think using `timestamptz` instead of `timestamp` is better here.
1. Because it is more consistent with `date_created` and `date_updated`
2. Because it makes the timestamp region independent (I think it can make a difference with dailight savings time and stuff, not sure though)

Reply:
I was seeing TZ converted to local, but I wonder if that was just a fluke. I found a good article on this topic and it essentially said to have the app code convert to UTC before insert and store the DB column as 'tz', always. To avoid TZ hell, IMO it is best to have the DB be strictly UTC going in and coming out.

In sum, I agree & updated: 16ae75e